samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -80,18 +90,10 @@ def training_input(input_name, s3_uri):
 def mnist_classification(
     region="us-east-1",
```

Comment:
We should default this to us-east-1

Reply:
fine with this since the whole sample is referring to running in us-east-1